Gems/ScriptCanvas/Code/scriptcanvasgem_headers.cmake

```diff
@@ -377,6 +371,11 @@ set(FILES
     Include/ScriptCanvas/Serialization/BehaviorContextObjectSerializer.h
     Include/ScriptCanvas/Serialization/DatumSerializer.h
     Include/ScriptCanvas/Serialization/RuntimeVariableSerializer.h
+    Include/ScriptCanvas/Data/BehaviorContextObject.h
+    Include/ScriptCanvas/Data/BehaviorContextObjectPtr.h
+    Include/ScriptCanvas/Data/Constants.h
+    Include/ScriptCanvas/Data/Data.h
+    Include/ScriptCanvas/Data/DataRegistry.h
```

Comment:
Should the other `Include/ScriptCanvas/Data` includes here be moved upwards instead?